### DISCUSSION TITLE
Viewsonic Woodpad PF1030

### DISCUSSION THREAD

**Discussion by haktanozcorekci** [OpenTabletDriver/OpenTabletDriver#2718] [Q&A] [answered]
There is pf0730 in the list of supported tablets, but not pf1030. Is there a possibility to add it?

**Comment by jamesbt365** [comment]
Support cannot be provided without an owner of the tablet stepping forward and assisting the developers with actually supporting the tablet. This is much better off done within our Discord, which you can find at https://opentabletdriver.net/discord

**Comment by gonX** [comment, marked as answer]
This will be tracked in #3040
